common/src/main/java/com/wynntils/wynn/model/map/poi/Poi.java

```diff
@@ -20,4 +20,24 @@ public MapLocation getLocation() {
     public abstract Texture getIcon();
 
     public abstract String getName();
+
+    /**
```

Comment:
Not entirely sure this belongs as part of a Poi. Looks more like a RenderUtils helper function to me; all it needs is the getLocation.getX/Z from the poi. It looks like there's a lot of rendering logic here, while the rest of the Poi class is just a "pure" representation of a location.